sf-stream/src/protos/extractor.proto

```diff
@@ -0,0 +1,290 @@
+// Copyright (c) Aptos
```

Comment:
Where do these proto files come from?

Reply:
we created this proto file; it's a requirement for the integration with StreamingFast